docker/Dockerfile-api

```diff
@@ -15,18 +15,19 @@ RUN dnf -y install \
     mod_auth_gssapi \
     mod_ssl \
     openssl-devel \
-    python3-devel \
-    python3-pip \
-    python3-wheel \
+    python38-devel \
+    python38-pip \
+    python38-wheel \
     && dnf update -y \
     && dnf clean all
+RUN update-alternatives --set python3 $(which python3.8)
 COPY . .
 COPY ./docker/iib-httpd.conf /etc/httpd/conf/httpd.conf
 
-# default python3-pip version for rhel8 python3.6 is 9.0.3 and it can't be updated by dnf
+# default python3-pip version for rhel8 python3.8 is 19.3.1 and it can't be updated by dnf
 # we have to update it by pip to version above 21.0.0
 RUN pip3 install --upgrade pip
-RUN pip3 install -r requirements.txt --no-deps --require-hashes
+RUN pip3 install -r requirements-py38.txt --no-deps --require-hashes
```

Comment:
Do we have to use pip3

Reply:
It was using `pip3` before my change and I kept it, the only difference is the requirements file and also that I used `update-alternatives` to ensure all commands goes to python 3.8 instead of the default on RHEL (3.6).
The names `python` and `pip` are generally not set, as you can see [here](https://developers.redhat.com/blog/2019/05/07/what-no-python-in-red-hat-enterprise-linux-8)